src/ontology/uberon-edit.obo

```diff
@@ -224513,6 +224513,18 @@ relationship: dc-contributor https://orcid.org/0000-0001-6677-8489 ! Aleix Puig-
 relationship: located_in UBERON:8600025 ! intervillous space
 property_value: dcterms-date "2024-04-15T10:05:35Z" xsd:dateTime
 
+[Term]
+id: UBERON:8900000
+name: sensory corpuscle
+def: "A mechanosensory complex in the terminal end of some nerves that forms a corpuscle. It consists of one or more neuronal terminations and associated non-neuronal cells." [PMID:35356056]
+synonym: "COEC" EXACT [PMID:35356056]
+synonym: "cutaneous end-organ complex" EXACT [PMID:35356056]
+xref: http://www.wikidata.org/entity/Q126363978
+is_a: UBERON:0005162 ! multi cell part structure
+is_a: UBERON:0012449 ! mechanoreceptor
+property_value: http://purl.org/dc/elements/1.1/date "2024-06-12T13:51:05Z" xsd:dateTime
```

Comment:
I think your date configuration is not correct and it is causing the error. In your setting, can you change it to: `http://purl.org/dc/terms/date`?
![image](https://github.com/obophenotype/uberon/assets/94959119/52013817-630d-45aa-8ff5-db28ca194998)